ui/src/ui/components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -0,0 +1,89 @@
+// Import packages
+import React, { useState } from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+// Importing react icons
+import { AiOutlineClose, AiOutlineUser } from 'react-icons/ai';
+
+// Import components
+import SpecialButton from '../SpecialButton/SpecialButton';
+import InputField from '../InputField/InputField';
+
+// import rtk query
+import { useForgotPasswordMutation } from '../../../store/api';
+
+// Import css
+import './ForgotPassword.css';
+import '../../../App.css';
+
+// Defining ForgotPassword Page
+const ForgotPassword = ({ setisModalOpen }: {setisModalOpen: React.Dispatch<React.SetStateAction<boolean>>}) => {
+  const navigate = useNavigate();
+  const [formData, setFormData] = useState({
+    email: '',
+  });
+  const { email } = formData;
+
+  const [forgotPassword] = useForgotPasswordMutation();
+
+  /**
+   * Function to send password reset email to the user on forgot password
+   */
+  const onForgotPassword = async () => {
+    const res = await forgotPassword(formData);
+    const response = ('data' in res ? res.data : 'data' in res.error ? res.error.data : null);
+
+    if (response.success) {
+      toast.success(`${response.message}`, {
+        duration: 3000,
```

Comment:
`3000` is this minutes or hours? write a comment against it.

Reply:
Done